packages/orchestrator-ui-components/src/types/subscription.ts

```diff
@@ -12,14 +12,32 @@ export enum SubscriptionStatus {
 export type Subscription = {
     subscriptionId: string;
     description: string;
-    note: string;
+    note?: string;
     startDate: string;
```

Comment:
Maybe we discussed this before (I forgot), but why is a date represented as a string and not a type?

Reply:
The actual type of the field being returned from the API is a string. It is shaped as a date (ISO). I do admit that we might want to move to 2 different types of "Subscription": one that is the actual shape of the API (startDate is string) and one that represents where the data actually stands for (startDate is a Date, id is a UUID etc). Currently we parse the date very close to the component itself